homeworks/lesia.turua_lesiaturuta/PokemonProject/scripts/pokemon.js

```diff
@@ -1,273 +1,21 @@
-            'Rock',
-        ],
-        number: '0004',
-        height: 24.0,
-        collectibles_slug: 'charmander',
-        featured: 'true',
-        slug: 'charmander',
-        name: 'Charmander',
-        ThumbnailAltText: 'Charmander',
-        ThumbnailImage: 'https://assets.pokemon.com/assets/cms2/img/pokedex/detail/004.png',
-        id: 4,
-        type: [
-            'fire',
-        ],
-    },
-    {
-        abilities: [
-            'Blaze',
-        ],
-        detailPageURL: '/us/pokedex/charmeleon',
-        weight: 41.9,
-        weakness: [
-            'Water',
-            'Ground',
-            'Rock',
-        ],
-        number: '0005',
-        height: 43.0,
-        collectibles_slug: 'charmeleon',
-        featured: 'true',
-        slug: 'charmeleon',
-        name: 'Charmeleon',
-        ThumbnailAltText: 'Charmeleon',
-        ThumbnailImage: 'https://assets.pokemon.com/assets/cms2/img/pokedex/detail/005.png',
-        id: 5,
-        type: [
-            'fire',
-        ],
-    },
-    {
-        abilities: [
-            'Blaze',
-        ],
-        detailPageURL: '/us/pokedex/charizard',
-        weight: 9999.0,
-        weakness: [
-            'Water',
-            'Electric',
-            'Rock',
-        ],
-        number: '0006',
-        height: 1102.0,
-        collectibles_slug: 'charizard',
-        featured: 'true',
-        slug: 'charizard',
-        name: 'Charizard',
-        ThumbnailAltText: 'Charizard',
-        ThumbnailImage: 'https://assets.pokemon.com/assets/cms2/img/pokedex/detail/006.png',
-        id: 6,
-        type: [
-            'fire',
-            'flying',
-        ],
-    },
-    {
-        abilities: [
-            'Drought',
-        ],
-        detailPageURL: '/us/pokedex/charizard',
-        weight: 221.6,
-        weakness: [
-            'Water',
-            'Electric',
-            'Rock',
-        ],
-        number: '0006',
-        height: 67.0,
-        collectibles_slug: 'charizard',
-        featured: 'true',
-        slug: 'charizard',
-        name: 'Charizard',
-        ThumbnailAltText: 'Charizard',
-        ThumbnailImage: 'https://assets.pokemon.com/assets/cms2/img/pokedex/detail/006.png',
-        id: 6,
-        type: [
-            'fire',
-            'flying',
-        ],
-    },
-    {
-        abilities: [
-            'Tough Claws',
-        ],
-        detailPageURL: '/us/pokedex/charizard',
-        weight: 243.6,
-        weakness: [
-            'Ground',
-            'Rock',
-            'Dragon',
-        ],
-        number: '0006',
-        height: 67.0,
-        collectibles_slug: 'charizard',
-        featured: 'true',
-        slug: 'charizard',
-        name: 'Charizard',
-        ThumbnailAltText: 'Charizard',
-        ThumbnailImage: 'https://assets.pokemon.com/assets/cms2/img/pokedex/detail/006.png',
-        id: 6,
-        type: [
-            'fire',
-            'dragon',
-        ],
-    },
-];
+let pokemons = [];
```

Comment:
Hey, @lesiaturuta, thanks for a good homework. 
Here is bug for you:
1. Type something in the search bar and hit enter
2. Choose a radio btn
Expected result: only pokemons matched the search text will be rendered
Actual result: all cards are shown.

![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/44c6dd53-cd3b-4592-a5ae-9500cc93aa98)